views/apps.ejs

```diff
@@ -5,7 +5,18 @@
     <link rel='stylesheet' href='/stylesheets/style.css' />
   </head>
   <body>
-    <h1><%= title %></h1>
-    <p>Welcome to <%= title %></p>
+    <div id="root"></div>
+
+
+    <!-- scripts -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.24/browser.js"></script>
```

Comment:
I think, we should be using 

```html
<script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
```

as it is mentioned in the babel docs. Lets try this !